sycl/doc/extensions/experimental/sycl_ext_oneapi_non_uniform_groups.asciidoc

```diff
@@ -666,34 +544,36 @@ public:
 
 [source,c++]
 ----
-id_type get_group_id() const;
+operator fragment<ParentGroup>() const;
 ----
-_Returns_: An `id` representing the index of the tangle-group.
+_Returns_: A `fragment` representing the same work-items as this chunk.
 
-NOTE: This will always be an `id` with all values set to 0, since there can
-only be one tangle-group.
+[source,c++]
+----
+id_type get_group_id() const;
+----
+_Returns_: An `id` representing the index of the chunk within its parent.
 
 [source,c++]
 ----
 id_type get_local_id() const;
 ----
 _Returns_: An `id` representing the calling work-item's position within
-the tangle-group.
+the chunk.
 
 [source,c++]
 ----
 range_type get_group_range() const;
 ----
-_Returns_: A `range` representing the number of tangle-groups.
-
-NOTE: This will always return a `range` of 1 as there can only be one
-tangle-group.
+_Returns_: A `range` representing the number of chunks created when the parent
+group was partitioned.
 
 [source,c++]
 ----
 range_type get_local_range() const;
 ----
-_Returns_: A `range` representing the number of work-items in the tangle-group.
+_Returns_: A `range` representing the number of work-items in the chunk, which
+is always equal to `ChunkSize`.
```

Comment:
Same comments as for `fragment` above:
- `get_local_liner_id` description is recursive
- `get_group_linear_range` and `get_local_linear_range` should return `linear_id_type`
- `get_group_linear_range` and `get_local_linear_range` descriptions should be "`range` returned by" instead of "`id` returned by"

Reply:
Fixed this above, in https://github.com/intel/llvm/commit/cc24795326e7da667c927e701ec472e93ec67d6b.